react-native-libraries.json

```
@@ -10505,5 +10505,14 @@
     "images": ["https://github.com/AnkiPro/react-native-window-resize/raw/main/assets/example.gif"],
     "ios": true,
     "dev": true
+  },
+  {
+    "githubUrl": "https://github.com/iChengbo/react-native-imagemin-asset-plugin",
+    "npmPkg": "react-native-imagemin-asset-plugin",
+    "examples": ["https://github.com/iChengbo/react-native-imagemin-asset-plugin/tree/master/playground"],
+    "images": ["https://github.com/iChengbo/react-native-imagemin-asset-plugin/blob/master/example.png"],
+    "ios": true,
+    "android": true,
+    "expoGo": true
```

Hello @iChengbo,, thank you for adding new entry to the directory! 👍 

Let's also add the [`dev` flag](https://github.com/react-native-community/directory?tab=readme-ov-file#dev) to the entry, since the package is a part of bundling process.

```suggestion
    "expoGo": true,
    "dev": true
```